### DISCUSSION TITLE
Can't plug three DS3 controllers

### DISCUSSION THREAD

**Discussion by bodzebod** [nefarius/DsHidMini#181] [Support]
Hi, I used to plug wirelessly 3 DS3 controllers on my PC with SCPToolkit and it was working like a charm. 
I switch to DsHidMini and BthPs3 since ScpToolkit is no longer working on my win11 (and I couldn't determine why actually, nevermind).
With DsHidMini, I manage to plug my controllers identified as "PLAYSTATION(R)3Conteroller-PANHAI". However, the led is not showing the controller identifier (used to be led 1 for controller 1, led2 for controller 2...), it's always showing either 3 or 4. 
While it's not a problem if I plug only one controller, it becomes an issue when plugging a second and a third. They also show either led 3 or 4, sometimes the first shuts down suddenly and no longer appairs, and I also have frequent BSOD errors related to wdf01000.sys (driver related). If I manage to plug two controllers, they seem to be identified as one unique controller: games are moving the same character with the two controllers.
Do you have any plans for supporting several controllers? Thank you.

**Comment by nefarius** [comment]
- [LEDs do not indicate player slots](https://github.com/ViGEm/DsHidMini#features) as [clearly documented](https://vigem.org/projects/DsHidMini/HID-Device-Modes-Explained/)
- The crash [is documented and will be addressed eventually](https://github.com/ViGEm/BthPS3/issues/19)
- Each controller is its own device and registered by the game as such, if not, configure your setup properly, again, all the help you need is in the linked docs
- There is no controller limit on DsHidMini whatsoever

Good luck!
